README.rst

```diff
@@ -1,6 +1,33 @@
 Cloudant-Python
 ===============
 
+This Version is Deprecated
+--------------------------
+
+As of 13 October 2015, this development branch is deprecated in favor of `the new 
+development branch starting at version 2.0.0a1 <https://github.com/cloudant/python-cloudant>`_. 
```

Comment:
Do we want to say "repo" rather than "branch" -- it implies the fact we're starting from scratch better than the term branch?